### PR TITLE
feat(trading): closed market tests

### PR DESCRIPTION
**Pull request by bwallacee**
Tests moved over from the monorepo: https://github.com/vegaprotocol/frontend-monorepo/pull/4660

We are missing two tests due to market-sim capabilities currently

### CODE REVIEW

File: tests/market/test_closed_markets.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        expected_pattern = re.compile(r"https://.*?/oracles/[a-f0-9]{64}")
+        actual_href = row_selector.locator('[col-id="settlementDataOracleId"] [data-testid="link"]').get_attribute("href")
+        assert expected_pattern.match(actual_href), f"Expected href to match {expected_pattern.pattern}, but got {actual_href}"
+        
+        # 6001-MARK-018
+        expect(row_selector.locator('[col-id="settlementAsset"]')).to_have_text("tDAI")
+        # 6001-MARK-020
+        expect(row_selector.locator('[col-id="settlementDate"]')).to_have_text("4 months ago")
+
+class TestTerminatedMarket:
```
Comment (daro-maj) [inline tests/market/test_closed_markets.py:110]: why do we need class ?

File: tests/market/test_closed_markets.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
+import pytest
+import re
+import vega_sim.api.governance as governance
+from playwright.sync_api import Page, expect
+from fixtures.market import setup_continuous_market
+from conftest import init_vega
+
+
+@pytest.fixture(scope="class")
+def vega():
```
Comment (daro-maj) [inline tests/market/test_closed_markets.py:10]: why do we need this ?

**Review by daro-maj — APPROVED** [review]

File: tests/market/test_closed_markets.py (hunk 2 above)
Comment (bwallacee) [reply tests/market/test_closed_markets.py:10]: This is so we can reuse the same instance of vega across each class of tests

File: tests/market/test_closed_markets.py (hunk 1 above)
Comment (bwallacee) [reply tests/market/test_closed_markets.py]: Removed

**Pull request merged**